include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -21,157 +24,808 @@ namespace NovelRT::Maths
         }
 
     public:
+        /**
+         * @brief The X component of the vector.
+         */
         float x;
+        /**
+         * @brief The Y component of the vector.
+         */
         float y;
 
+        /**
+         * @brief Creates a GeoVector2F with all components set to zero.
+         *
+         * @return A GeoVector2F with all components set to zero.
+         */
         GeoVector2F() noexcept : x(0.0f), y(0.0f)
         {
         }
+
+        /**
+         * @brief Creates a GeoVector2F with the given components.
+         *
+         * @param x The X component of the vector.
+         * @param y The Y component of the vector.
+         * @return A GeoVector2F with the given components.
+         */
         GeoVector2F(float x, float y) noexcept : x(x), y(y)
         {
         }
 
+        /**
+         * @brief Constructs a unit GeoVector2F with a magnitude of one by normalizing this vector.
```

Comment:
@RubyNova - might be a nit, but is `normalizing` supposed to be `normalising` in UK English?

Reply:
I believe so.

Reply:
```suggestion
         * @brief Constructs a unit GeoVector2F with a magnitude of one by normalising this vector.
```